### PR TITLE
[WIP] fixed bugs with mvvm support

### PR DESCRIPTION
**Pull request by f1nzer**
#12
- [x] restrict item to be in IsSwipedLeft and IsSwipedRight state at the same time
- [x] correctly sync IsSwipedLeft <--> IsSwipedRight values
- [x] fix for IsSwipedLeft <--> IsSwipedRight transition (ignoring None state)
- [ ] handle case when ViewModel.IsSwiped\* == true at startup (also handle this case on ListViewItem reusing)

p.s. also applied some coding styles and some properties moved to C#6

### CODE REVIEW

File: LLMListView/LLMListViewItem.cs [hunk 1]
```diff
@@ -349,7 +384,7 @@ protected override void OnManipulationDelta(ManipulationDeltaRoutedEventArgs e)
             }
         }
 
-        void SwipeToLeft(double cumulativeX, double deltaX)
+        private void SwipeToLeft(double cumulativeX, double deltaX)
```
Comment (f1nzer) [inline LLMListView/LLMListViewItem.cs:389]: @brookshi cumulativeX value passed to this method is not used. Can we remove it from here?

File: LLMListView/SwipeReleaseAnimationConstructor.cs [hunk 2]
```diff
@@ -182,7 +182,6 @@ public void SwipeTo(SwipeDirection direction, SwipeConfig config, bool animated)
             else
             {
                 config.MainTransform.X = targetX;
```
Comment (f1nzer) [inline LLMListView/SwipeReleaseAnimationConstructor.cs:184]: Have you any ideas why it's equal to 0 even when it's defined OnLoad, etc? You can try OnLoad set _mainTransform to any value but it's reseted to 0 anyway. :bow:

File: LLMListView/LLMListViewItem.cs (hunk 1 above)
Comment (brookshi) [inline LLMListView/LLMListViewItem.cs:389]: yes, it's redundant, we should remove it.

File: LLMListView/SwipeReleaseAnimationConstructor.cs (hunk 2 above)
Comment (brookshi) [inline LLMListView/SwipeReleaseAnimationConstructor.cs:184]: @F1nZeR  Curiously, I also don't know why. Only visible item's transform will be reset to 0, you can find the other items is ok if you scroll the list. 
ContentPresenterTranslateTransform is origin ListViewItem style property, it seems that ContentPresenterTranslateTransform is changed when render the visible items.

**Pull request merged**